src/dataexplorer/dataexplorer-application/src/components/SchemaResponse/Schemes/Schemes.tsx

```diff
@@ -0,0 +1,281 @@
+    answers: IAnswer[];
+    questionnaireResource: fhir.ValueSet[];
+    date: Dayjs;
+    title: string;
+}
+
+const Schemes = (props: SchemesProps) => {
+    dayjs.locale('nb');
+    const [qAndA, setQAndA] = useState<IQuestionAndAnswer[]>([]);
+    useEffect(() => {
+        let hasAddedId = false;
+        // eslint-disable-next-line
+        props.questions.map((q) => {
+            // eslint-disable-next-line
+            props.answers.map((a) => {
+                if (q.id === a.id) {
+                    hasAddedId = true;
+                    setQAndA((qAndA) => [
+                        ...qAndA,
+                        { id: q.id, questions: q, answers: a },
+                    ]);
+                }
+            });
+            hasAddedId === false &&
+                setQAndA((qAndA) => [...qAndA, { id: q.id, questions: q }]);
+            hasAddedId = false;
+        });
+    }, [props]);
+
+    const { Panel } = Collapse;
+
+    const setContent = (data: fhir.ValueSetComposeIncludeConcept[]) => {
+        if (data) {
+            return data.map((line) => <li key={line.code}>{line.display}</li>);
+        }
+        return;
+    };
+
+    return (
+        <>
+            <Row justify="center">
+                <div className="card">
+                    <h1 className="title">{props.title}</h1>
+                    <Row justify="center">
+                        {props.date.format('DD/MM/YYYY HH:mm')}
+                    </Row>
+                    <Collapse
+                        bordered={false}
+                        style={{ width: 1000 }}
+                        className="site-collapse-custom-collapse"
+                    >
+                        {qAndA.map(
+                            (section) =>
+                                section.id.split('.').length === 1 && (
+                                    <Panel
+                                        header={
+                                            section.questions.questions.text
+                                        }
+                                        key={section.id}
+                                        className="site-collapse-custom-panel"
+                                    >
+                                        {qAndA.map(
+                                            (qa, qaIndex) =>
+                                                qa.id.split('.').length === 2 &&
+                                                section.id ===
+                                                    qa.id.split('.')[0] &&
+                                                qa.answers?.answers.answer &&
+                                                qa.answers?.answers
+                                                    .answer[0] && (
+                                                    <div
+                                                        key={qaIndex}
+                                                        className="boarder"
```

Comment:
Is this a typo?

Reply:
Don't know, this is @rebekkbb's scss classes